packages/rspack/tests/configCases/css/urls/webpack.config.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
What is this file for?

Reply:
make sure this test are tested, if it doesn't have webpack.config.js it will be skipped

Reply:
actually this tests should be put in caseTest other than configTest, since it doesn't change any configuration